extras/asm-multicluster/README.md

```diff
@@ -0,0 +1,133 @@
+
+```
+kubectx cluster-1 
+kubectl apply -f services-all.yaml 
+kubectl apply -f ../../kubernetes-manifests/config.yaml
+kubectl apply -f ../../extras/jwt/jwt-secret.yaml
+kubectl apply -f ../../kubernetes-manifests/accounts-db.yaml 
+kubectl apply -f ../../kubernetes-manifests/contacts.yaml
+kubectl apply -f ../../kubernetes-manifests/frontend.yaml
+kubectl apply -f ../../kubernetes-manifests/userservice.yaml
+kubectl apply -f ../../kubernetes-manifests/loadgenerator.yaml
+kubectl apply -f ../../istio-manifests/frontend-ingress.yaml
+
+kubectx cluster-2 
+kubectl apply -f services-all.yaml 
+kubectl apply -f ../../kubernetes-manifests/config.yaml
+kubectl apply -f ../../extras/jwt/jwt-secret.yaml
+kubectl apply -f ../../kubernetes-manifests/ledger-db.yaml
+kubectl apply -f ../../kubernetes-manifests/balance-reader.yaml
+kubectl apply -f ../../kubernetes-manifests/ledger-writer.yaml
+kubectl apply -f ../../kubernetes-manifests/transaction-history.yaml
+```
+
+5. **Verify that the pods start up successfully.** Note that you should see 2 containers per pod `(2/2)`, one containing the Bank of Anthos service container, the other containing the ASM sidecar proxy (Envoy). 
+
+```
+kubectx cluster-1; kubectl get pods 
+```
+
+You should see: 
+
+```
+NAME                             READY   STATUS    RESTARTS   AGE
+accounts-db-0                    2/2     Running   0          14m
+contacts-848bbff9bd-bjtrh        2/2     Running   0          14m
+frontend-565f986f97-r5tdq        2/2     Running   0          14m
+loadgenerator-797dfd6c5c-r86vn   2/2     Running   0          14m
+userservice-7c9bc855f5-ff8tq     2/2     Running   0          14m
+```
+
+Do the same for cluster-2: 
+
+```
+kubectx cluster-2; kubectl get pods 
+```
+
+You should see: 
+
+```
+NAME                                  READY   STATUS    RESTARTS   AGE
+balancereader-5f57666b9c-l9q4b        2/2     Running   0          14m
+ledger-db-0                           2/2     Running   0          14m
+ledgerwriter-7d88d5898c-vhs5l         2/2     Running   0          14m
+transactionhistory-84cf479f65-lzgdt   2/2     Running   0          14m
+```
+
+6. **Access the Bank of Anthos frontend** using the Istio IngressGateway's LoadBalancer IP. 
+
+```
+kubectx cluster-1; kubectl get svc istio-ingressgateway -n istio-system 
+
+NAME                   TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)                                                                      AGE
+istio-ingressgateway   LoadBalancer   10.7.254.103   <EXTERNAL_IP>  15021:30429/TCP,80:32101/TCP,443:31503/TCP,15012:30573/TCP,15443:31131/TCP   21m
+```
+
+Navigate to the `EXTERNAL_IP` in a browser; you should see the Bank of Anthos login page. 
+
+7. **Open the Google Cloud Console, and navigate to Anthos > Servic Mesh**. View the Bank of Anthos services in the Anthos Service Mesh dashboard. In the Table view, you should see metrics populated for services in both cluster-1 and cluster 2. 
```

Comment:
At this point I had to `Enable` Anthos when I navigated to the page in the console. Should we mention it here?

![image](https://user-images.githubusercontent.com/7249208/113190782-bb161380-922a-11eb-8b80-0d8324891597.png)